pipeline/data_merging/test_aggregate_across_columns.py

```diff
@@ -1,5 +1,6 @@
 import pytest
 import unittest
+import pdb
```

Comment:
can remove pdb before merging since it's no longer needed (it's best to have fewer imports)